UnitTests/GitUITests/CommandsDialogs/FormAddSubmoduleTests.cs

```diff
@@ -34,6 +34,12 @@ public void SetUp()
             _gitExecutable = Substitute.For<IExecutable>();
         }
 
+        [OneTimeTearDown]
+        public void OneTimeTearDown()
+        {
+            _gitExecutable = null;
```

Comment:
This should not be necessary. If it is necessary, it should be in a `TearDown` method not a `OneTimeTearDown` method.